Sources/Plasma/PubUtilLib/plSurface/CMakeLists.txt

```diff
@@ -19,7 +18,6 @@ set(plSurface_HEADERS
     plLayerAnimation.h
     plLayerDepth.h
     plLayerInterface.h
-    plLayerMultiply.h
```

Comment:
This file wasn't deleted?

Reply:
It doesn't [seem to exist](https://github.com/Hoikas/Plasma/tree/vcperf/Sources/Plasma/PubUtilLib/plSurface/plLayerMultiply.h)?

Reply:
Looks like the deletion is shown in the commit view for 52f736d61e9a4c714d8820b49a650a5eb3a7985f

Reply:
I see, git seems to think you renamed it pfConditional/Pch.h  :facepalm:

Reply:
I am suspicious of those being the same file... :trollface:

Reply:
🚬